docs/utilities/batch.md

```diff
@@ -250,30 +250,48 @@ Inheritance is importance because we need to access message IDs and sequence num
 
 === "SQS"
 
-    ```python hl_lines="8 17 27 34"
+    ```python hl_lines="8 17 27 35"
     --8<-- "examples/batch_processing/src/pydantic_sqs.py"
     ```
 
+=== "SQS - Sample Event "
+
+    ```json hl_lines="6 22"
+    --8<-- "examples/batch_processing/src/pydantic_sqs_event.json"
+    ```
+
```

Comment:
Added an event payload because this payload is very specific and must be a dict to make Pydantic work.